### PR TITLE
Quick fix to an exception occuring on 1.17.0 stable and 1.18.0 beta

### PR DESCRIPTION
**Pull request by JohnMurga**
There is probably a cleaner way to do this, but the fix seemed safe enough

### CODE REVIEW

**Comment by JohnMurga** [comment]
Apologies, testing revealed that my quick fix was maybe too quick ...
Should correctly guard for when the underlying component becomes null now.
Which for me is on multiple quick temp tab closes

**Comment by JohnMurga** [comment]
Added fix for #38 on activation in Atom 1.19.0+

File: lib/highlight-column-element.coffee [hunk 1]
```diff
@@ -52,12 +52,13 @@ class HighlightColumnView extends HTMLDivElement
   updateHighlight: ->
     if @isEnabled()
       rect = @highlightRect()
-      width = rect.width
-      width = 1 if @isLineMode()
-      @style.left = "#{rect.left}px"
-      @style.width = "#{width}px"
-      @style.opacity = @opacity()
-      @style.display = 'block'
+      if rect
```
Comment (tony612) [inline lib/highlight-column-element.coffee]: I refer to return early when rect is null like
```
return if rect
```
and in other similar places.

File: lib/highlight-column-element.coffee [hunk 2]
```diff
@@ -73,19 +74,30 @@ class HighlightColumnView extends HTMLDivElement
...
+        rect
 
   _cursorPixelRect: ->
     {row, column} = @cursor.getScreenPosition()
     screenRange = new Range(new Point(row, column), new Point(row, column + 1))
-    rect = @editorElement.pixelRectForScreenRange(screenRange)
-    range = @editorElement.pixelRangeForScreenRange(screenRange)
-    rect.left = range.start.left
-    rect.right = range.end.left
+    if  @editorElement.component != null
```
Comment (tony612) [inline lib/highlight-column-element.coffee]: and here

**Comment by tony612** [comment]
ping @JohnMurga

File: lib/highlight-column-element.coffee (hunk 1 above)
Comment (JohnMurga) [inline lib/highlight-column-element.coffee]: Not sure if important to do the ```@style.display = 'none'``` before return here

File: lib/highlight-column-element.coffee (hunk 2 above)
Comment (JohnMurga) [inline lib/highlight-column-element.coffee]: Here I have a feeling that was important to explicitly return rect as ```null```

**Comment by JohnMurga** [comment]
Really sorry @tony612, I had never done a review trough here, so I wasn't sure what to do ...
Would you like me to merge latest and apply the changes according to your feedback ?

**Pull request merged**